playbooks/roles/topbeat/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Download topbeat 1.0.0
+  get_url:
+    url: "https://download.elastic.co/beats/topbeat/topbeat_1.0.0_amd64.deb"
+    dest: "/tmp"
+
+- name: Install topbeat 1.0.0
+  apt:
+    deb: "/tmp/topbeat_1.0.0_amd64.deb"
+
+- name: Install httplib2
+  pip:
+    name: httplib2
+
+- name: Enable topbeat at boot
+  service:
+    name: topbeat
+    enabled: yes
+  notify:
+    - restart topbeat
+
+- name: Ensure {{ topbeat_app_dir }} exists
```

Comment:
Per previous discussion, I think we decided to not worry about symlinking into `/edx/app/`?